tools/src/faircv/Main.hs

```diff
@@ -96,7 +96,8 @@ main = do
 
     Just proof <- return $ mkMerkleProof tree (Set.fromList [0..9])
 
-    fcv :: FairCV <- return $ unReadyFairCV $ singletonFCV addr (hash hdr) (readyProof proof)
+    fcv :: FairCV <- return $ unReadyFairCV $
+        singletonFCV addrS "John Doe" addr (hash hdr) (readyProof proof)
```

Comment:
Such classique :doge:

Reply:
But hey, this line was unironically too long - 94 chars is way more than the recommended 80 (and close to the hard limit of 100).